test/optimizedForceMove/_acceptableWhoSignedWhat.test.ts

```diff
@@ -0,0 +1,69 @@
+import {ethers} from 'ethers';
+import {expectRevert} from 'magmo-devtools';
+// @ts-ignore
+import optimizedForceMoveArtifact from '../../build/contracts/TESTOptimizedForceMove.json';
+import {setupContracts} from './test-helpers';
+
+const provider = new ethers.providers.JsonRpcProvider(
+  `http://localhost:${process.env.DEV_GANACHE_PORT}`,
+);
+let optimizedForceMove: ethers.Contract;
+
+const participants = ['', '', ''];
+const wallets = new Array(3);
+
+// populate wallets and participants array
+for (let i = 0; i < 3; i++) {
+  wallets[i] = ethers.Wallet.createRandom();
+  participants[i] = wallets[i].address;
+}
+
+beforeAll(async () => {
+  optimizedForceMove = await setupContracts(provider, optimizedForceMoveArtifact);
+});
+
+describe('_acceptableWhoSignedWhat (expect a boolean)', () => {
+  it.each`
+    whoSignedWhat | largestTurnNum | nParticipants | nStates | expectedResult
+    ${[0, 1, 2]}  | ${2}           | ${3}          | ${3}    | ${true}
+    ${[0, 1, 2]}  | ${5}           | ${3}          | ${3}    | ${true}
+    ${[0, 0, 1]}  | ${2}           | ${3}          | ${2}    | ${true}
+    ${[0, 0, 0]}  | ${2}           | ${3}          | ${1}    | ${true}
+    ${[0, 0, 0]}  | ${8}           | ${3}          | ${1}    | ${true}
+    ${[0, 0, 2]}  | ${2}           | ${3}          | ${3}    | ${false}
+    ${[0, 0, 2]}  | ${11}          | ${3}          | ${3}    | ${false}
```

Comment:
Might be nice to add a column with a brief description of what the test case covers e.g. "n states" or "<n states" etc.

Reply:
Addressed in https://github.com/statechannels/nitro-protocol/pull/15/commits/62cbd0f0e797b5cd51a94e70e401f70d414fed52